napalm/iosxr.py

```diff
@@ -84,16 +84,22 @@ def rollback(self):
     def get_facts(self):
 
         sh_ver = self.device.show_version()
-        match_sh_ver = re.search('Cisco IOS XR Software, Version (.*)\nCopyright .*\n(.*) uptime is (.*)\nSystem .*\n(.* Series) ', sh_ver, re.DOTALL)
-        os_version = match_sh_ver.group(1)
-        hostname = match_sh_ver.group(2)
-        fqdn = match_sh_ver.group(2)
-        uptime = string_parsers.convert_uptime_string_seconds(match_sh_ver.group(3))
-        model = match_sh_ver.group(4)
-        serial_number = None
 
-        # todo
-        interface_list = []
+        for line in sh_ver.splitlines():
+            if 'Cisco IOS XR Software' in line:
+                os_version = line.split()[-1]
+            elif 'uptime' in line:
+                uptime = string_parsers.convert_uptime_string_seconds(line)
```

Comment:
This can't be on the entire "line" as they look like this: 
iosrx-napalm.test.host uptime is 33 minutes 
we need everything after: $hostname uptime is...